colossalai/utils/profiler/common.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+from pathlib import Path
+from typing import List, Union
+
+from colossalai.core import global_context as gpc
+
+from .comm_profiler import CommProfiler
+
+
+class ProfilerActivity(Enum):
+    '''
+    Event types to profile
+    '''
+    COMM = CommProfiler
+
+
+class colossalai_profiler(object):

```

Comment:
Follow python class naming convention

Reply:
we can directly do this

```python
with Profiler(...) as prof:
    do_something()
```

I don't think we need to have the repository name in the class name.

Reply:
Sure, but conventionally context classes are lowercase

Reply:
I see, I can accept the current code then.